properties/genre.rb

```diff
@@ -9,5 +9,6 @@ def initialize(name)
 
   def add_item(item)
     @items << item
+    item.genre(self)
```

Comment:
Yes, well done on all changes I made a mistake myself on this one it is ` item.genre = self` and not `item.genre(self)`